hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchService.kt

```diff
@@ -64,11 +64,13 @@ class AttributeSearchService(
     return elasticsearchClient.search(searchRequest)
       .also {
         telemetryMap["resultCount"] = it.hits.hits.size.toString()
-        telemetryMap["totalHits"] = it.hits.totalHits?.toString() ?: "0"
-        telemetryMap["timeInMs"] = it.took?.toString() ?: "0"
+        telemetryMap["totalHits"] = it.hits.totalHits?.toString()?.extractInt() ?: "0"
+        telemetryMap["timeInMs"] = it.took?.toString()?.extractInt() ?: "0"
```

Comment:
Previously looked like this: 
![image](https://github.com/ministryofjustice/hmpps-prisoner-search/assets/58170926/0ccc796d-b569-4f76-9ad6-a3d016319de6)